web_src/js/modules/tippy.js

```diff
@@ -94,6 +96,7 @@ function attachTooltip(target, content = null) {
 
 function switchTitleToTooltip(target) {
   let title = target.getAttribute('title');
+  if (target.getAttribute('data-no-tooltip')) return;
```

Comment:
No use?

Reply:
TBH, I see how you use it, but I have objection for doing so.

Do not show too many texts on the UI, and do not abuse relative-time and/or title/tooltip

Reply:
I've removed it, turns out setting title to empty string results in `<relative-time>` not setting any title, which is what I want here.